modal/stub.py

```diff
@@ -508,6 +510,35 @@ def _add_function(self, function: _Function) -> _FunctionHandle:
     def registered_functions(self) -> List[str]:
         return list(self._function_handles.keys())
 
+    @property
+    def registered_entrypoints(self) -> Dict[str, Callable]:
+        return self._local_entrypoints
+
+    def local_entrypoint(self, raw_f=None, name: Optional[str] = None):
+        """Decorate a function to be used as a CLI entrypoint for a Modal App
+
+        These functions can be used to do initialization of apps using local
+        assets. Note that regular Modal functions can also be used as entrypoints,
+        but unlike local_entrypoint Modal function are executed remotely.
+
+        E.g.
+        @stub.local_entrypoint
+        def main():
+            some_modal_function()
+
+        To run the function, either call it directly from code:
+        ```python
+        main()  # needs no stub.run() context
+        ```
+        Or from the CLI:
+        ```
+        modal app run stub_module.py::main
```

Comment:
you don't need `:main` if there's just one entrypoint right?

Reply:
Good point, I need to fix that!

Reply:
And the syntax here needs to be updated, since it's `modal run file_or_module[::stub_name] [function_name]` where the `[]` are optional